app/src/ui/repository-settings/fork-settings.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react'
+import { DialogContent } from '../dialog'
+import { ForkContributionTarget } from '../../models/workflow-preferences'
+import { RepositoryWithForkedGitHubRepository } from '../../models/repository'
+
+interface IGitIgnoreProps {
+  readonly forkContributionTarget: ForkContributionTarget
+  readonly repository: RepositoryWithForkedGitHubRepository
+  readonly onForkContributionTargetChanged: (
+    forkContributionTarget: ForkContributionTarget
+  ) => void
+}
+
+enum RadioButtonId {
+  Parent = 'ForkContributionTargetParent',
+  Self = 'ForkContributionTargetSelf',
+}
+
+/** A view for creating or modifying the repository's gitignore file */
+export class ForkSettings extends React.Component<IGitIgnoreProps, {}> {
+  public render() {
+    return (
+      <DialogContent>
+        <h2>I'll be using this fork…</h2>
+
+        <div className="radio-component">
+          <input
+            type="radio"
+            id={RadioButtonId.Parent}
+            value={ForkContributionTarget.Parent}
+            checked={
+              this.props.forkContributionTarget ===
+              ForkContributionTarget.Parent
+            }
+            onChange={this.onForkContributionTargetChanged}
+          />
+          <label htmlFor={RadioButtonId.Parent}>
+            To contribute to the parent repository
+          </label>
+        </div>
+        <div className="radio-component">
+          <input
+            type="radio"
+            id={RadioButtonId.Self}
+            value={ForkContributionTarget.Self}
+            checked={
+              this.props.forkContributionTarget === ForkContributionTarget.Self
+            }
+            onChange={this.onForkContributionTargetChanged}
+          />
+          <label htmlFor={RadioButtonId.Self}>To use for my own purposes</label>
```

Comment:
The grammar here is a bit messy, what about something like:
```suggestion
          <label htmlFor={RadioButtonId.Self}>For my own purposes</label>
```

Reply:
Thanks for the suggestion!

Since I'm not a native speaker I'll leave this to @ampinsk and @billygriffin .

Context of this text:

![image](https://user-images.githubusercontent.com/408035/82656787-96bdd380-9c24-11ea-9e9d-120b8c4f5037.png)

Reply:
"For my own purposes" makes sense to me!